packages/scss/src/components/tableSorted/component.scss

```diff
@@ -0,0 +1,52 @@
+@use '@lucca-front/scss/src/components/button/exports' as button;
+@use '@lucca-front/icons/src/icon/exports' as icon;
+
+@mixin component($atRoot: 'without: rule') {
+	@include button.text;
+	@include button.withIcon;
+
+	--components-button-padding: 0 var(--pr-t-spacings-50);
+	--components-button-gap: var(--pr-t-spacings-25);
+
+	user-select: none;
+	font-weight: 400;
+	vertical-align: top;
+	align-items: flex-start;
+	left: var(--components-sorted-offset);
+	text-align: inherit;
+	flex-direction: var(--components-sorted-direction);
+
+	@at-root ($atRoot) {
+		.table-head-row-cell-sortableButton-arrows,
```

Comment:
If we consider tableSorted as a component, maybe could replace `.table-head-row-cell-sortableButton` and `.indexTable-head-row-cell-sortableButton` basis by `.tableSortable` ?